src/test/test_step.go

```diff
@@ -34,6 +35,30 @@ var numUploadFailures int64
 
 const maxUploadFailures int64 = 10
 
+var targetsTested = metrics.NewCounter(
+	"test",
+	"targets_tested_total",
+	"Total number of targets tested",
+)
+
+var targetsCached = metrics.NewCounter(
+	"test",
+	"targets_cached_total",
+	"Total number of targets not re-tested due to caching",
+)
```

Comment:
I think this includes both those cached in `plz-out` and in the dir/http caches. Seems difficult to report those differently in the way this code is set out. I think however we report this, we should probably align that with how we report for building.